src/cuga/backend/server/main.py

```
@@ -854,6 +854,13 @@ def _get_draft_include_by_app():
     yield
     logger.info("Application is shutting down...")
 
+    from cuga.backend.storage import get_storage
+
+    try:
+        await get_storage().close_relational_stores()
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)